inventory/group_vars/all

```diff
@@ -0,0 +1,15 @@
+tng_user: tango
+tng_pass: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          38663035396632396335386233613565613137383933313931386261303461343736663138653131
+          6261643731653734623063353264643866633738313633330a353865343938306632346564313861
+          37393539363661656661633330616231396566366663623564666265613661663431653765306632
+          3862376532643831370a646464373434363161333132386264663537333632633865643563393431
+          3634
+domain_name: sonata-nfv.eu
+fqdn: "{{ plat }}.{{ pop }}.{{ domain_name }}"
```

Comment:
I think fqdn is not more plat.pop.domain. This is an example: pre-int-sp.5gtango.eu so it will be environment-plat-domain_name. Are you agreee?

Reply:
right, this a decision up to the TNG steering commitee: having a single domain "5gtango.eu" but not sub-domains like:
* sp.alabs.5gtango,eu, * vnv.ncsrd.5gtango,eu, * sdk.bcn5gtango,eu
to be fixed